Assets/MirageXR/ContentTypes/VirtualInstructor/Editors/SpatialUI/SpatialUiAiTextToSpeechModel.cs

```diff
@@ -42,6 +42,8 @@ public override void Initialization(Action<PopupBase> onClose, params object[] a
             _objectDataSet = RootObject.Instance.LEE.ArtificialIntelligenceManager.GetTtsModels();
             InstantiateObjectData(_objectDataSet);
             close.onClick.AddListener(Close);
+            TryToGetArguments(args);
```

Comment:
you shouldn't call `TryToGetArguments(args);`
it will be called from `base.Initialization(onClose, args);`
```
public virtual void Initialization(Action<PopupBase> onClose, params object[] args)
    {
        _onClose = onClose;
        if (!TryToGetArguments(args))
        {
            Debug.LogError("error when trying to get arguments!");
        }
    }
```

Reply:
done